conn.go

```diff
@@ -207,13 +206,12 @@ func (c *Conn) RDY() int64 {
 
 // LastRDY returns the previously set RDY count
```

Comment:
I guess this function is somewhat deprecated now?

Reply:
Yea